packages/relay/tests/lib/eth.spec.ts

```diff
@@ -3390,17 +3390,17 @@ describe('Eth calls using MirrorNode', async function () {
         }
       });
 
-      sdkClientStub.submitContractCallQueryWithRetry.returns({
+      /*sdkClientStub.submitContractCallQueryWithRetry.returns({
```

Comment:
Did you mean to remove these comments?

Reply:
Also when you don't expect a fallback to consensus not would be good to ensure this is not called

Reply:
✅  done